src/test/java/pro/sky/questions/QuestionServiceTest.java

```diff
@@ -0,0 +1,71 @@
+package pro.sky.questions;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import pro.sky.questions.exceptions.NotFoundQuestionsException;
+import pro.sky.questions.model.Question;
+import pro.sky.questions.service.QuestionServiceClass;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+
+public class QuestionServiceTest {
+    QuestionServiceClass questionServiceClass = new QuestionServiceClass();
+
+    @Test
+    void testAdd() {
+        questionServiceClass.add("pen", "write");
+        questionServiceClass.add("book", "read");
+        questionServiceClass.add("weather", "cold");
+        assertEquals(3, questionServiceClass.getAll().size());
+        assertThat(questionServiceClass.getAll()).containsExactlyInAnyOrder(
+                new Question("summer", "sun"),
+                new Question("winter", "snow"),
+                new Question("rain", "water")
+        );
```

Comment:
Тут ты проверяешь, что все добавленные вопросы присутсвуют в списке вопросов, однако тебе нужно проверять те же вопросы, которые ты добавил. Поэтому должно быть примерно так:
```suggestion
        assertThat(questionServiceClass.getAll()).containsExactlyInAnyOrder(
                new Question("pen", "write"),
                new Question("book", "read"),
                new Question("weather", "cold")
        );
```